### PR TITLE
Fix issue#130

### PR DESCRIPTION
**Pull request by hishamco**
@Rick-Anderson I added this based on your suggestion before on #130, nothing but I comment the lines, because it's not needed unless someone want to you the SQL Server

### CODE REVIEW

**Comment by aspnet-hello** [comment]
This issue was moved to aspnet/Home#2397

**Comment by aspnet-hello** [comment]
Sorry I was a bad bot. Re-opening this PR.

**Comment by Eilon** [comment]
@muratg - any further action planned on this PR?

**Comment by muratg** [comment]
@Tratcher could you take a look when you have a little bit of time?

File: samples/SessionSample/Startup.cs [hunk 1]
```diff
@@ -5,13 +5,31 @@
...
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 
 namespace SessionSample
 {
     public class Startup
     {
+        // Uncomment the following line to use the Microsoft SQL Server implementation of IDistributedCache.
```
Comment (Tratcher) [inline samples/SessionSample/Startup.cs]: Can't you leave all of this constructor uncommented? It doesn't change the app behavior. Only AddSqlServerCache needs to be commented out.

File: samples/SessionSample/Startup.cs [hunk 2]
```diff
@@ -13,36 +13,33 @@ namespace SessionSample
...
 
-            // Uncomment the following line to use the Microsoft SQL Server implementation of IDistributedCache.
-            // Note that this would require setting up the session state database.
-            //services.AddSqlServerCache(o =>
-            //{
-            //    o.ConnectionString = Configuration["AppSettings:ConnectionString"];
-            //    o.SchemaName = "dbo";
-            //    o.TableName = "Sessions";
-            //});
+            services.AddSqlServerCache(o =>
```
Comment (Tratcher) [inline samples/SessionSample/Startup.cs]: Sorry, I meant for this section to stay commented out, this is the only part that changes app behavior.

Comment (hishamco) [reply samples/SessionSample/Startup.cs]: Yep, but this need the constructor to be commented out too

Comment (Tratcher) [reply samples/SessionSample/Startup.cs]: Why? All the constructor does is create a config object, that's harmless.

**Comment by muratg** [comment]
@Tratcher could you take another look? Let's either move it forward or close it.

**Comment by hishamco** [comment]
@muratg I updated the PR to avoid the conflict. Waiting for @Tratcher review to either take it or break it 📆 ⌚️

**Comment by Tratcher** [comment]
The config changes are fine, but AddDistributedMemoryCache should still be the default and AddSqlServerCache should remain commented out.

**Comment by Tratcher** [comment]
Thanks.

**Pull request merged**